src/MSBuild/JsonOutputFormatter.cs

```diff
@@ -0,0 +1,139 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Text.Json.Nodes;
+using Microsoft.Build.Evaluation;
+using Microsoft.Build.Execution;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Shared;
+
+namespace Microsoft.Build.CommandLine
+{
+    internal class JsonOutputFormatter
+    {
+        private static readonly JsonSerializerOptions Options = new() { AllowTrailingCommas = true, WriteIndented = true };
```

Comment:
I don't know why this is true, but I added this because I also like trailing commas, then tested it and found that the commas didn't show up? I don't know why but I didn't fight it...I can make it explicitly false, though.